chessmate/engines.py

```diff
@@ -179,19 +179,20 @@ def __init__(self):
         self.name = "Prioritize Bishop Moves"
 
     def evaluate(self, board: chess.Board) -> None:
-        """ Assigns same value to each move since eventually going to choose
-        random move """
+        """ Assigns values to each move depending on if
+        capturing is involved """
         self.reset_move_variables()
 
         legal_move_list = list(board.legal_moves)
         for m in legal_move_list:
             if get_piece_at(board, str(m)[:2].upper()) == "B":
-                if board.is_capture(m):
+                piece_at_position = get_piece_at(board, str(m)[2:4]).upper()
```

Comment:
I'm having a hard time conceptualizing what this is doing. I may have to revisit this after we wrap things into a generic function